### ISSUE TITLE
Update planter registration data, tree capture upload files to have better names

### ISSUE THREAD

**Issue by arunbakt** [Greenstand/treetracker-android#555]
The current files that are uploaded to s3 buckets from tree tracker app uses filenames with md5 hash of the json file appended with system date-time value.  This doesn't help in easily identifying the type of data contained in the uploaded files as planter registration versus planter tree captures data.  

Update the files names to contain a prefix such as `planter_info` or `tree_captures` or other appropriate text based on the file being uploaded. This would be useful generally to troubleshoot issues in production.

**Comment by deepwinter** [comment]
Yes I have been thinking we need this, thank you for bringing this up.
